client/state/ui/guided-tours/actions.js

```diff
@@ -68,3 +74,15 @@ export function nextGuidedTourStep( { tour = 'main', stepName } ) {
 
 	return withAnalytics( trackEvent, nextAction );
 }
+
+// TODO(mcsf): come up with a much better (read: safer) solution
```

Comment:
Can you extend the comment to include what you think the proble with this solution is?